### PR TITLE
Refactor!

### LINKED ISSUE #23: import.js filepath is not taken.
When I run import.js in bash (exec folder is 'openaddresses-master', PELIAS_CONF set to my conf file)

The importer searches the file in the execution folder instead of getting the datapath path set in the config

my config:
```
"openaddresses": {
      "datapath": "\\\\prod\\..(morepath)...\\openaddresses-collected\\be\\wa\\",
      "files": ["brussels-fr.csv"]
    }
```

I guess smth wrong happens here as the "filePath" variable should be the absolute path, but it contains only the file name.

```
logger.info( 'Importing %s files.', files.length );
  files.forEach( function forEach( filePath ){
    recordStream.append( function ( next ){
      logger.info( 'Creating read stream for: ' + filePath );
      next( importPipelines.createRecordStream( filePath ) );
    });
  });
```

### PR DESCRIPTION
**Pull request by orangejulius**
This PR includes all of #31 and #33 plus...

* move all the importer steps to their own module with a consistent interface
* switch to using passthrough streams when admin lookup or deduplication are disabled to make the logic cleaner (h/t to @dianashk for the example from pelias/openstreetmap)
* Add a functional test of the entire pipeline from two real OA files

Fixes pelias/pelias#228
Fixes #23

### CODE REVIEW

File: lib/importPipeline.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *        https://github.com/pelias/address-deduplicator-stream
+ *
+ *      adminValues: Add admin values to each address object (since
+ *        OpenAddresses doesn't contain any) using `admin-lookup`. See the
+ *        documentation: https://github.com/pelias/admin-lookup
+ */
+function createFullImportPipeline( files, opts, finalStream ){
+  logger.info( 'Importing %s files.', files.length );
+
+  finalStream = finalStream || elasticsearchStream.create();
```
Comment (dianashk) [inline lib/importPipeline.js]: this is a cool maneuver. great for testing.

Comment (orangejulius) [inline lib/importPipeline.js]: Stole a page from the book of @trescube

File: lib/streams/adminLookupStream.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var through = require( 'through2' );
+var logger = require( 'pelias-logger' ).get( 'openaddresses' );
+var peliasAdminLookup = require( 'pelias-wof-admin-lookup' );
+
+function createAdminLookupStream(enable, peliasConfig) {
+  if (enable) {
+    logger.info( 'Setting up admin value lookup stream.' );
```
Comment (dianashk) [inline lib/streams/adminLookupStream.js]: here you log inside the `if`, but for deduper outside. should probably be consistent. i like having the log statement inside the `if` so it's muted when disabled.

**Comment by dianashk** [comment]
This is looking really great now. Code is easy to follow and testable/tested. Other than the minor logging statement comment, it looks ready to go.

:ship:

File: lib/streams/adminLookupStream.js (hunk 2 above)
Comment (orangejulius) [inline lib/streams/adminLookupStream.js]: oh good catch. logging was meant to go in the if statement in both cases

**Pull request merged**